cmd/genfeaturegates/genfeaturegates.go

```diff
@@ -0,0 +1,626 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/version"
+	"k8s.io/component-base/featuregate"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	_ "k8s.io/kubernetes/pkg/features"
```

Comment:
do we also need to `_` import these:

`k8s.io/apiserver/pkg/features`
`k8s.io/apiextensions-apiserver/pkg/features`